### DISCUSSION TITLE
Moving from RemNote to Obsidian

### DISCUSSION THREAD

**Discussion by cesar-programs** [st3v3nmw/obsidian-spaced-repetition#339] [General]
Thank you for creating this plug-in.

Obsidian's use of markdown and the user having the ability to manipulate the files however they want gives it so much power—I love having the ability to use macOS' Spotlight to search for relevant information in my markdown files from anywhere on the computer. Nevertheless, I never seemed to develop a solid and clean SR system (Anki plug-ins never seemed to fully work for me) so I had to stick with RemNote, not feeling secure about the longevity of my notes. 

This plug-in gave me the incentive to use Obsidian again and I am thrilled (Obsidian Vim is great). I look forward to its advancements and I believe it can be up to par with RemNote's flashcard implementation in time. 

With great appreciation and encouragement, 23727

**Comment by st3v3nmw** [comment]
I'm glad that you found the plugin useful!
